blocks/test/full-content.js

```diff
@@ -93,110 +81,51 @@ describe( 'full post content fixture', () => {
 		require( 'blocks' );
 	} );
 
+	it( 'has 54 fixtures', () => {
```

Comment:
It might be obsolete, but we can give it a try. Whenever we add/remove file, we would have to update this number.